targettests/execution/conditional_sample.cpp

```diff
@@ -6,13 +6,8 @@
  * the terms of the Apache License 2.0 which accompanies this distribution.    *
  ******************************************************************************/
 
-// clang-format off
-// RUN: nvq++ -std=c++17 --enable-mlir %s -o %t
-// RUN: if [ $(echo %cpp_std | cut -c4- ) -ge 20 ]; then \
-// RUN:   nvq++ --enable-mlir %s -o %t && %t; \
-// RUN: fi
-// RUN: nvq++ %cpp_std --target quantinuum --emulate %s -o %t && %t
-// clang-format on
+// REQUIRES: c++20
+// RUN: nvq++ --target quantinuum --emulate %s -o %t && %t
```

Comment:
Why was the test case `nvq++ --enable-mlir %s -o %t && %t` removed?

Reply:
It was wrapped in the odd `if` construct. I'll add it back.